plugins/uefi-capsule/efi/generate_binary.py

```diff
@@ -95,6 +93,15 @@ def _run_objcopy(args):
             ".dynsym",
             "-j",
             ".rel*",
+        ]
+        if args.sbat_distro_id:
+            argv += [
+                "-j",
+                ".sbat",
+                "--set-section-alignment",
+                ".sbat=512",
+            ]
```

Comment:
I think we always want to do the explicit alignment and not only when sbat_distro_id is defined, but I let @vathpela and @ chrisccoulson to confirm that’s the case.

Reply:
I moved the `-j .sbat` too

Reply:
Ah, right. Looks good to me.